something/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
```

Comment:
Please do not create this "something". Instead, add a few test cases inside `frb_example/pure_dart`

Reply:
this is just for fast testing the generation of StreamSink without creating a flutter project, not going to be in the final PR either

Reply:
Btw, the `examples/pure_dart` example is a pure-dart example, no need for flutter project as well